src/react/styled.ts

```diff
@@ -46,14 +44,14 @@ function styled(tag: React.ComponentType<*> | string) {
       }
     }
 
-    const render = (props, ref) => {
+    const render = (props: any, ref: any) => {
```

Comment:
something like this maybe?
```suggestion
    const render = (props: { [key: string]: unknown }, ref: React.RefObject<unknown>) => {
```
not sure if ref object is different for function refs and createRefs

Reply:
I have no idea what the correct types are for this file

Reply:
let's leave it for later then, no worries. marking a `// TODO: better types` would do just fine :)

Reply:
Added a `// TODO` comment for whole `styled` component